operator/docs/user-guides/recover_ingester_pods_in_zone_failure_outages.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Playbook for recovering ingester pods in zone-failure outages"
+description: "Playbook for recovering ingester pods in zone-failure outages"
+lead: ""
+date: 2023-07-25T08:48:45+00:00
+lastmod: 2023-07-25T08:48:45+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "user-guides"
+weight: 100
+toc: true
+---
+
+This document describes how to recover the ingester pods in the event of a zone failure.
+
+_Disclaimer:_ This document describes a recovery procedure by manually recreating the failed pods in another zone. Right now, we are doing this by deleting PVCs of the impacted pods from the failed zone, so it can be recreated in a different zone. This will cause data loss of the data in the PVC. To avoid actual data loss we always set the replication factor in loki to be 1 or higher so data is always replicated.
```

Comment:
nit
```suggestion
_Disclaimer:_ This document describes a recovery procedure by manually recreating the failed pods in another zone. Right now, we are doing this by deleting PVCs of the impacted pods from the failed zone, so they can be recreated in a different zone. This will cause data loss of the data in the PVC. To avoid actual data loss we always set the replication factor in loki to be 1 or higher so data is always replicated.
```

Reply:
Done